styles.css

```diff
@@ -0,0 +1,366 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+ul li {
+  list-style: none;
+  margin-left: 20px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: 1fr 4fr;
+  grid-template-rows: 1fr 4fr;
+  height: 100vh;
+}
+
+.side-bar {
+  color: white;
```

Comment:
you should try and make use of RGB, hex and so on. For example white is #fff, try to be making use of them